VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-quick-link/copy-quick-link.component.html

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Is this missing [attr.alt] like in the other one? [attr.alt]="'copy-telephone-id.alt' | translate"?